helm/templates/metric-anomaly-task-manager/deployment.yaml

```diff
@@ -78,16 +78,16 @@ spec:
               mountPath: /var/{{ .Chart.Name }}/rules/alert-rules.json
               subPath: alert-rules.json
           livenessProbe:
-            initialDelaySeconds: {{ int .Values.livenessProbe.initialDelaySeconds }}
-            periodSeconds: {{ int .Values.livenessProbe.periodSeconds }}
+            initialDelaySeconds: {{ int .Values.metricAnomalyTaskManagerConfig.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ int .Values.metricAnomalyTaskManagerConfig.livenessProbe.periodSeconds }}
```

Comment:
This is outside the scope of this PR, but switching to real probes rather than these TCP probes is something we should work to do on any new service. Series of PRs where I added to attribute service:
https://github.com/hypertrace/attribute-service/pull/100
https://github.com/hypertrace/attribute-service/pull/101
https://github.com/hypertrace/attribute-service/pull/102

Reply:
https://github.com/hypertrace/hypertrace/issues/319